ansible/meta/main.yml

```diff
@@ -13,5 +13,5 @@ galaxy_info:
     - name: Ubuntu
       versions:
         - focal
-
+  role_name: bare_metal_paperless
```

Comment:
How is this related?

Reply:
its my try on fixing the ansible related checks for the PR. (Although currently that did not help.)

Reply:
reverted those changes as those ansible tests are much more complex than expected.